src/domain/community/virtualContributor/vcProfilePage/VCProfileContentView.tsx

```
@@ -154,3 +202,34 @@ export const VCProfileContentView = ({
 
 export default VCProfileContentView;
+
+function SectionTitle({ children }) {
+  return (
+    <BlockTitle display={'flex'} alignItems={'center'} gap={gutters(0.5)}>
+      {children}
+    </BlockTitle>
+  );
+}
+
+function SectionWrapper({ children }: PropsWithChildren<{}>) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)